docs/docs/sources/chatplugin/customization.md

```diff
@@ -2,3 +2,62 @@
 title: Basic & advanced customization
 sidebar_label: Basic & advanced customization
 ---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+## How to customize the Airy Chat Plugin
```

Comment:
Before we get into the paragraphs we should introduce the two concepts: we have two ways of configure airy chat plugin. For most use cases... basic blaa bla bla. If you need full control blah blah blah :)